catalog/app/containers/Bucket/Summary.js

```diff
@@ -187,16 +209,18 @@ export default function BucketSummary({
   )
   const { readme, images, summarize } = extractSummary(files)
 
+  const canAddReadme = false // !readme && !!packageHandle AND can open RevisePackage with url
```

Comment:
this is some stub i guess

Reply:
Yes, I'm going to uncomment it after https://github.com/quiltdata/quilt/pull/2957 merge